src/components/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import "../Footer.scss"
+
+export function Footer() {
```

Comment:
I think we're preferring the arrow function notation here:
```suggestion
export const Footer: React.FC = () => {
```
(you will need to import React at the top of the file)